templates/travel/components/ActivitiesPanel.js

```diff
@@ -0,0 +1,62 @@
+import ActivityButton from './ActivityButton';
+import ActiveSvg from './svg/ActiveSvg';
+import ArtisticSvg from './svg/ArtisticSvg';
+import DrinksSvg from './svg/DrinksSvg';
+import FashionSvg from './svg/FashionSvg';
+import FoodSvg from './svg/FoodSvg';
+import MusicSvg from './svg/MusicSvg';
+import NatureSvg from './svg/NatureSvg';
+import NightlifeSvg from './svg/NightlifeSvg';
+import ToursSvg from './svg/ToursSvg';
+import WaterSvg from './svg/WaterSvg';
+
+export default function ActivitiesPanel(props) {
```

Comment:
Your other examples put the export default at the end. Do you want the AMP Start examples to follow that approach, or use this style of export on the function declaration? (This is for me to do on AMP Start demos, not for you to change here as this file will go away later as ampstart project will hold master copy.)

Reply:
I think your approach is good. Not sure why I did it differently.